chef-utils/lib/chef-utils/dist.rb

```diff
@@ -88,8 +126,15 @@ class Solo
     end
 
     class Workstation
+      # The full marketing name of the product
+      PRODUCT = "Chef Workstation"
+
       # The suffix for Chef Workstation's /opt/chef-workstation or C:\\opscode\chef-workstation
       DIR_SUFFIX = "chef-workstation"
+
+      # Workstation banner/help text
+      DOCS = "https://docs.chef.io/workstation/"
+      HEADER = "The Chef command line tool for managing your infrastructure from your workstation."
```

Comment:
Since this is only used in a single place can't we just use the data we already have here in the CLI help to print out a friendly message?

Reply:
Yeah, I see that it's used just here:

https://github.com/chef/chef-cli/blob/77bb4462183db64f8677cbc6b884b85f91004aa1/lib/chef-cli/cli.rb#L36

And can probably use one of the other constants.